### PR TITLE
Small fixes, mostly to vars and f strings

### PR DESCRIPTION
**Pull request by micziz**
# Related Issue/Addition to code
Added some small fixes, just fixing some things. Mostly esthetically

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Proposed Changes
In global variables, use just one. In multiple strings, do not use multiple prints, but use triple """. Also small fixes to comments

# Additional Info
Python is designed to be efficient, and global variables help it. It also helps in preformance. Triple """ is better for readability and ese of use. Just plain old fixes for comments

## Checklist:

- [x] My code follows the style guidelines of this project and have read CONTRIBUTING.md
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by micziz** [comment]
Not ready

**Comment by micziz** [comment]
Chcecks made, cfr

**Review by compsup — APPROVED** [review]
LGTM. Loving the new PR's addressing these small inconsistencies and non-pythonic ways about doing things.

**Comment by EthanHindmarsh** [comment]
> LGTM. Loving the new PR's addressing these small inconsistencies and non-pythonic ways about doing things.

Yes, other than the `print`s (which I might be wrong about), this PR looks great ✅

**Comment by ThioJoe** [comment]
A couple of the indentations / alignments on new lines aren't there, also you forgot to remove some of the `',")` at the end of some lines

File: YTSpammerPurge.py [hunk 1]
```diff
@@ -20,18 +20,16 @@
...
+###              same directory as this script. See the Readme for instructions on this.
 ###
 ###           2. I suck at programming so if something doesn't work I'll try to fix it but might not
 ###              even know how, so don't expect too much.
 ###
-### Author:   ThioJoe - YouTube.com/ThioJoe
-###                     Twitter.com/ThioJoe
+### Author:   ThioJoe - YouTube.com/ThioJoe Twitter.com/ThioJoe
 ###
 ### GitHub:   https://github.com/ThioJoe/YT-Spammer-Purge/
```
Comment (KendallDoesCoding) [inline YTSpammerPurge.py:30]:
```suggestion
### GitHub:   https://github.com/ThioJoe/YT-Spammer-Purge
```

**Review by KendallDoesCoding — COMMENTED** [review]
This would be a good fix too, the url works even without the / in the end of it!

File: YTSpammerPurge.py [hunk 2]
```diff
@@ -286,7 +284,8 @@ class MiscDataStore:
 
 
   #----------------------------------- Begin Showing Program ---------------------------------
-  print(f"{F.LIGHTYELLOW_EX}\n===================== YOUTUBE SPAMMER PURGE v" + version + f" ====================={S.R}")
+  print(f"{F.LIGHTYELLOW_EX}\n===================== YOUTUBE SPAMMER PURGE v{version} ====================={S.R}")
```
Comment (KendallDoesCoding) [inline YTSpammerPurge.py:287]:
```suggestion
  print(f"{F.LIGHTYELLOW_EX}\n===================== YT-SPAMMER-PURGE v{version} ====================={S.R}")
```

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]

File: YTSpammerPurge.py (hunk 2 above)
Comment (TechStudent11) [reply YTSpammerPurge.py:287]: YT = YouTube

I think it should remain YOUTUBE SPAMMER PURGE without the dashes. Idk, maybe that's me.

Comment (EthanHindmarsh) [reply YTSpammerPurge.py:287]: > I think it should remain YOUTUBE SPAMMER PURGE without the dashes. Idk, maybe that's me.
YT-Spammer-Purge is the repository name, it's best to remain consistent in my opinion 😁

File: YTSpammerPurge.py [hunk 3]
```diff
@@ -54,7 +52,7 @@
 from Scripts.utils import choice
 
 print("Importing Standard Libraries...")
-# Standard Libraries
+#Import Standard Libraries
```
Comment (EthanHindmarsh) [inline YTSpammerPurge.py:55]:
```suggestion
# Import Standard Libraries
```

**Review by EthanHindmarsh — CHANGES_REQUESTED** [review]
Tiny change😅

**Pull request closed without merging**